examples/tutorials/Introduction_to_FerminetElectronFeature_layer.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<p>Let's install torch geometric and deepchem together, it gives a cleaner look</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3931/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>